### ISSUE TITLE
TOC links to sections with special characters don't work.

### ISSUE THREAD

**Issue by nachogt** [BoostIO/Boostnote#3461]
# Current behavior

When a section of the note (line starting with one or more # characters) has a special character in it, the links generated in the TOC don't work for that section.

For example, if I have a section called:

## Conceptos básicos

the TOC generates a link called "#Configuracion-basica" (without the á character). When clicked, a jump to that section does not happen. The rest of links to other sections do work, though.

I noticed this when clicking on links generated on an HTML exported note. The links on the TOC of the HTML don't work either for sections with at least these special characters in it (so, maybe that's two issues in one).

# Expected behavior

Clicking on a TOC link should jump to that section.

# Steps to reproduce

1. Create a section with a special character in it 
2. Create the TOC
3. Click on the link

# Environment

- Version : 0.14.0
- OS Version and name : MacOS Catalina

**Comment by Flexo013** [comment]
Strange, this does seem to work fine on Windows 10, with 0.14.0.

![image](https://user-images.githubusercontent.com/4507015/74610005-a1317400-50ef-11ea-98b4-24d6f5201238.png)


Could somebody with Mac double check if this issue is indeed Mac only?

**Comment by ignaciogarcia** [comment]
I created a new note to replicate the problem and indeed, it worked correctly now.

Went back to the note originating the issue, and as described, it was not working. Updated the TOC and then everything started to working.

Maybe it was a problem of a previous version (the note was from over a year ago) and was fixed at some point. Sorry I didn't do this check before... I think the issue could be closed now.

Thank you and again sorry for the confusion.
